src/wallet/swapcoin.rs

```diff
@@ -46,7 +57,18 @@ pub struct IncomingSwapCoin {
     pub hash_preimage: Option<Preimage>,
 }
 
-/// Represents an outgoing swapcoin.
+/// Describes an outgoing swapcoin, which can either be currently active or successfully completed.
+///
+/// ### NOTE:
+/// The term `Outgoing` might imply that the swapcoin is still active or in progress.
+/// However, within this protocol, it specifically refers to a coin
+/// that was initially sent by a participant during a swap.
+///
```

Comment:
```suggestion
/// ### NOTE:
/// The term `Outgoing` imply an Outgoing Coin from a swap.
/// This can be for either a Taker or a Maker, depending on their position in the swap route.
/// This refers to coins that have been "sent" by the party, i.e. the party holds the time lock side of the contract.
///
```